mobile/src/actions/index.js

```diff
@@ -49,7 +50,7 @@ export const registerDevice = (router, location) => {
         clientParams: {
           redirectURI: 'http://localhost',
           softwareID: 'io.cozy.mobile.files',
-          clientName: 'Mobile Cozy Files',
+          clientName: genId(),
```

Comment:
I do not want to add [](https://github.com/apache/cordova-plugin-device) to get `cordova.device.model`.

But without any plugin we can have something like `${cordova.platformId} (${genId()})` (in a template string) to have something like: **android (5ogdneqea9e)**. What do you think about, @aenario?